Tip to set seed.

juhiarora9 opened the pull request:
PSU Bootcamp lesson exercise

juhiarora9 committed:
Tip to set seed.

jdblischak commented:
I can't automatically merge this because you used the same filename that I did. Could you please run the following 3 lines in the terminal to update your pull request?

```
git mv set-seed.md set-seed-2.md
git commit -m "fix filename conflict"
git push origin master
```

juhiarora9 commented:
I'm so sorry!!
I ran the 3 lines.
Here's the output. Is this what you're looking for?
[image: image.png]


On Fri, Aug 14, 2020 at 12:05 PM John Blischak <notifications@github.com>
wrote:

> I can't automatically merge this because you used the same filename that I
> did. Could you please run the following 3 lines in the terminal to update
> your pull request?
>
> git mv set-seed.md set-seed-2.md
> git commit -m "fix filename conflict"
> git push origin master
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jdblischak/computing-tips/pull/18#issuecomment-674146358>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIYC55VKCH6UGTF5WLQEZZTSAVOGLANCNFSM4P7UJWTA>
> .
>


-- 
Thank you,
Juhi Arora
Graduate Assistant,
Department of Veterinary and Biomedical Sciences.
Executive Mentor,
Graduate Women in Science.
Pennsylvania State University
University Park, PA (USA)

jdblischak commented:
Sorry I can't see that image. Can you try directly copy-pasting it into GitHub (not your email)?

The pull request hasn't been updated. Can you try pushing again with `git push origin master`?

juhiarora9 commented:
I tried to re-run the commands again.
This is the output I am getting:
[jua268@comp-lc-0079 computing-tips]$ git mv set-seed.md set-seed-2.md
fatal: not under version control, source=set-seed.md, destination=set-seed-2.md
[jua268@comp-lc-0079 computing-tips]$ git commit -m "fix filename conflict"
U	set-seed.md
error: 'commit' is not possible because you have unmerged files.
hint: Fix them up in the work tree,
hint: and then use 'git add/rm <file>' as
hint: appropriate to mark resolution and make a commit,
hint: or use 'git commit -a'.
fatal: Exiting because of an unresolved conflict.

jdblischak commented:
Hmmm....I'm not sure what is going on. Could you please send me the output of running the following?

```
git status
git log --stat
```

juhiarora9 commented:
Sure..

[jua268@comp-lc-0079 computing-tips]$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

You have unmerged paths.
  (fix conflicts and run "git commit")

juhiarora9 commented:
[jua268@comp-lc-0079 computing-tips]$ git log --stat
commit 1b4a2f5207c92b45997392f633a50806c881260f
Author: Juhi Arora <arora.juhi94@gmail.com>
Date:   Fri Aug 14 11:35:53 2020 -0400

    Tip to set seed.

 set-seed.md | 2 ++
 1 file changed, 2 insertions(+)

commit 6986dc01c34d0fa58d0e1facb217bc92530743b8
Merge: 8e51ad6 45a574c
Author: John Blischak <jdblischak@gmail.com>
Date:   Sat Aug 8 23:50:46 2020 -0400

    Merge pull request #1 from workflowr/master
    
    Added tip to use version control.

commit 45a574cbe6d4aafebf2716c7eb2b20ebc852bc14
Author: John Blischak <jdblischak@gmail.com>
Date:   Sat Aug 8 23:24:50 2020 -0400

    Added tip to use version control.

jdblischak commented:
So `git status` doesn't tell you which paths are the problem? Is there any information after this line? I would expect it to tell you with files were causing the problem.

```
You have unmerged paths.
(fix conflicts and run "git commit")
```

juhiarora9 commented:
Changes to be committed:

	new file:   1.sh
	new file:   CompTip.txt
	new file:   TheBestTip.md
	new file:   aliases.txt
	new file:   jmy5455.txt
	new file:   loop-creating.md
	new file:   make-comments.md
	new file:   tips.txt

Unmerged paths:
  (use "git add <file>..." to mark resolution)

	both added:      set-seed.md

jdblischak commented:
OK. Now I understand. There is a merge conflict. That's tougher to fix without me being able to see exactly what you see. If you're interested in fixing this, let's plan on troubleshooting together at the beginning of the next break.

juhiarora9 commented:
Okay, I will stay on for the next break. Thanks for your help!

juhiarora9 committed:
Merge branch 'master' of https://github.com/jdblischak/computing-tips

Conflicts:
	set-seed.md

juhiarora9 committed:
Tip to delete line in nano
